### ISSUE TITLE
Typo, maybe rephrase this paragraph a bit.

### ISSUE THREAD

**Issue by adrianvoica** [riot/riot.github.io#303]
I think the paragraph at "Case note - Textarea and value" (https://riot.js.org/documentation/#case-note---textarea-and-value) needs a bit of rephrasing, it's a bit convoluted as it is right now.

Also, there's (at least) a typo "It againsts" -> "It's against" ?

And, since we're at it, I think it would be appropriate to type DOM instead of "dom" in the documentation, just for clarity, since it's an acronym. If Mozilla calls it that way in the MDN, so can we :) (https://developer.mozilla.org/en-US/docs/Web/API/Document_Object_Model/Introduction)

Thanks.

**Comment by GianlucaGuarini** [comment]
@adrianvoica yes it makes sense. Go for it. PRs are welcome
